src/plone/restapi/deserializer/blocks.py

```diff
@@ -12,20 +12,34 @@
 from zope.publisher.interfaces.browser import IBrowserRequest
 
 
-def path2uid(context, path):
+def path2uid(context, portal, href):
```

Comment:
@cekk do we pass a full URL "href" to path2uid? If we name the function "path2uid" one of the input params should be "path" in my opinion.

Reply:
Just checked: it seems that volto returns a full url. In that method i try to manage both cases (full url or path).
Let me know what naming do you prefer and i'll fix it ;)